src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -23,11 +28,54 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public virtual void ConfigureServices(IServiceCollection services)
         {
             services.AddDevelopmentIdentityProvider(Configuration);
 
+            services
+                .AddRouting()
+
+                // Adding the GraphQL server core service
+                .AddGraphQLServer()
+
+                // Adding our scheme
+                .AddDocumentFromFile("./patient.graphql")
```

Comment:
Can these files live in Core\Data\GraphQl folder (same as we have `search-parameters.json`?